gpiozero/boards.py

```diff
@@ -1532,6 +1532,38 @@ def __init__(self, pwm=False, pin_factory=None):
         )
 
 
+class TrafficpHAT(LEDBoard):
```

Comment:
This class should extend `TrafficLights` not `LEDBoard`. Usage would just be `phat.on()` etc and `phat.green.on()` etc.

Reply:
Believe I've done this right now.